README.md

```diff
@@ -1,75 +1,38 @@
-# Coalesce
-Designed to help you quickly build amazing sites, Coalesce is a rapid-development code generation framework for 
-  * ASP.NET Core
-  * MVC Core
-  * EF Core
-  * TypeScript
-  * Visual Studio 2017
+# Coalesce  
 
-|master|dev (MyGet)
-|:--:|:--:
-|[![NuGet](https://img.shields.io/nuget/v/IntelliTect.Coalesce.svg)](https://www.nuget.org/packages/IntelliTect.Coalesce)|[![MyGet](https://img.shields.io/myget/intellitect-coalesce/v/IntelliTect.Coalesce.svg)](https://www.myget.org/feed/intellitect-coalesce/package/nuget/IntelliTect.Coalesce)
-|[![Build status](https://ci.appveyor.com/api/projects/status/mev829igrspj4x5s/branch/master?svg=true)](https://ci.appveyor.com/project/IntelliTect/coalesce/branch/master)|[![Build status](https://ci.appveyor.com/api/projects/status/mev829igrspj4x5s/branch/dev?svg=true)](https://ci.appveyor.com/project/IntelliTect/coalesce/branch/dev)
+[Home](http://coalesce.intellitect.com/) &middot; [Documentation](http://coalesce.readthedocs.io) &middot; [Get Started](#Get-Started) &middot; [Builds](#Builds)
 
-## Bottom Line
-Projects that used to take us three weeks now take one week. 
+Coalesce is a framework for rapid-development of ASP.NET Core web applications. It works from the Entity Framework Core data model that you design, automating the creation of the glue - DTOs, API Controllers, and TypeScript - that sit between your data and the UI of your application. 
 
-## Learn More
-You can find a sample app with documentation here: http://coalesce.intellitect.com/
+## Fundamentals
 
-## Get Started Fast
-We recommend using our starter project at at: https://github.com/IntelliTect/Coalesce.Starter
+* **Code Generation** Write your data model, and we'll build the DTOs, API controllers, and TypeScript that are needed to start building a front-end UI right away. A full suite of CRUD endpoints are created for you, with inner-workings that are overridable to great degrees.
```

Comment:
You can omit "to great degrees"

Reply:
Thanks - I have a tendency to add fluff words all over the place, but I wanted to keep them out of this as much as possible.

Reply:
"A full suite of CRUD endpoints are created for you, with inner-workings that are overridable to great degrees."
Is a bit choppy. Does "A full suite of CRUD endpoints with overridable inner-workings are available for your use" make sense?

Reply:
There's an extra at before the link under getting started fast.